### PR TITLE
docs(readme): add distro packages section

### PR DESCRIPTION
**Pull request by orhun**
Hello!

I packaged `watchbind` for Arch & Alpine Linux and this PR simply updates README.md for adding the installation instructions.

Thanks! :bear:

### CODE REVIEW

**Comment by fritzrehde** [comment]
Cool, that's awesome! I personally use Arch Linux too, and was thinking of trying to get this into the AUR at some point as well, but hadn't gotten around to doing so because I've never published anything to any Linux repositories.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,31 @@ The [releases page](https://github.com/fritzrehde/watchbind/releases) contains p
...
 ```
 
+### Distro Packages
+
+<details>
+  <summary>Packaging status</summary>
+
+[![Packaging status](https://repology.org/badge/vertical-allrepos/watchbind.svg)](https://repology.org/project/watchbind/versions)
+
+</details>
```
Comment (fritzrehde) [inline README.md:47]: One small thing: I actually like the repology packaging status icon, and I don't really see a reason to hide it behind a `<details>`.

Comment (orhun) [reply README.md]: Updated in 251e985b4e1f318ba26cc1448de6dbc8521bfd83

**Comment by fritzrehde** [comment]
LGTM! Thanks!

**Pull request merged**